dunwoody_disaster/FightSequence.py

```diff
@@ -90,6 +90,7 @@ def evaluateEnemyTurn():
 
         def finishTurn():
             playerActions.clear()
+            enemyActions.clear()
             enemyActions.selectRandom()
```

Comment:
Thank you Mr. bug hunter!
Maybe in `selectRandom` it should call `clear`?

Reply:
Well someone was kind enough to point out where it was likely happening. S/O to that individual! 

I could make that change if needed.